### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Modern auth enabling steps for 2013

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ Older email applications like Office 2013 need app passwords. Here's how to allo
     People can then use client Office apps after they create a new password.
     
 5. Select**Save**, then **Close**.
+
+> [!NOTE]
+> Modern authentication can be enabled On Office 2013 by setting a few registry Keys, see [Enable Modern Authentication for Office 2013 on Windows devices](https://docs.microsoft.com/office365/admin/security-and-compliance/enable-modern-authentication?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
> Modern authentication can be enabled On Office 2013 by setting a few registry keys. For more information, see [Enable Modern Authentication for Office 2013 on Windows devices](https://docs.microsoft.com/office365/admin/security-and-compliance/enable-modern-authentication?view=o365-worldwide).
```

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #672

Thank you.

**Comment by CrystalThomasMS** [comment]
@sirkkuw, reviewed and tweaked slightly. Good to merge?

**Comment by Sirkkuw** [comment]
@mypil I am ok with this.

**Pull request merged**